src/actions/GraphDataActions.ts

```diff
@@ -91,7 +99,7 @@ export const GraphDataActions = {
 
   // action creator that performs the async request
   fetchGraphData: (
-    namespace: Namespace,
+    namespaces: Namespace[],
```

Comment:
Perhaps we should have a Namespaces type instead of having to add the array brackets.

Reply:
I'm not sure I understand, `Namespace[]` is already a type (array of Namespaces). Unless you mean we might need to add data (now or in the future) apart of the array of namespaces.

Reply:
Yeah, maybe I was getting confused with Golang, never mind.